alethzero/MainWin.cpp

```diff
@@ -859,9 +859,8 @@ void Main::readSettings(bool _skipGeometry)
 	{
 		p->readSettings(s);
 	});
-
-	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setAsk(u256(s.value("askPrice", "500000000000").toString().toStdString()));
-	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setBid(u256(s.value("bidPrice", "500000000000").toString().toStdString()));
+	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setAsk(u256(s.value("askPrice", QString::fromStdString(boost::lexical_cast<std::string>(c_defaultGasPrice))).toString().toStdString()));
+	static_cast<TrivialGasPricer*>(ethereum()->gasPricer().get())->setBid(u256(s.value("bidPrice", QString::fromStdString(boost::lexical_cast<std::string>(c_defaultGasPrice))).toString().toStdString()));
```

Comment:
`toString` would be a shorter and more codebase-consistent version of `boost::lexical_cast<std::string>`